lcls-twincat-motion/Library/POUs/Motion/FB_MotionStage.TcPOU

```diff
@@ -42,7 +42,7 @@ stMotionStage.bLimAllBwd := stMotionStage.bLimBwd AND (stMotionStage.bLimGntBwd
 
 // Handle standard commands using ESS's FB
 fbDriveVirtual(En:=TRUE,
-	bEnable:=stMotionStage.bEnable,
+	bEnable:=stMotionStage.bEnable AND stMotionStage.bSTO,
```

Comment:
We need to refactor this a bit to allow an external function block with custody of `MC_Power` to correctly enable/disable without needing to learn how to correctly handle `bSTO`, like what we did with the limits. Also, we may consider renaming `bSTO` to make it more obvious that `bSTO = TRUE` is what is needed to allow motion.

Reply:
Reading the variable name as-is: `"boolean Safe-Torque-Off = True" means Enable`

:+1: on the rename.

Reply:
`bHardwareEnable`?

I just realized that all existing stages without hardware STO buttons will need to lock this variable to `TRUE`, but I guess that's ok? Also anything with a redundant STO won't be able to link to this variable directly, e.g. `M1.bSTO := bSTO1 AND bSTO2`, but I guess that's ok too

Reply:
`bHardwareEnable` sounds fine.

@ZLLentz I think that's just a detail that users will have to work around.